v0/destinations/recurly/data/RCAddressConfig.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "destKey": "address.phone",
+    "sourceKeys": "phone",
+    "sourceFromGenericMap": true
+  },
+  {
+    "destKey": "address.street1",
+    "sourceKeys": ["traits.address.street", "context.traits.address.street"]
+  },
+  {
+    "destKey": "address.city",
+    "sourceKeys": ["traits.address.city", "context.traits.address.city"]
+  },
+  {
+    "destKey": "address.region",
+    "sourceKeys": ["traits.address.state", "context.traits.address.state"]
+  },
+  {
+    "destKey": "address.postal_code",
+    "sourceKeys": [
+      "traits.address.postalCode",
+      "context.traits.address.postalCode"
+    ]
+  },
+  {
+    "destKey": "address.country",
+    "sourceKeys": ["traits.address.country", "context.traits.address.country"]
```

Comment:
check if not sent in iso code format it is still accepted.

Reply:
Yes it is still accepted. Recurly converts the country to ISO format and sends the response.
Request: 
{
    "code": "123456-1251",
    "email": "email--927@gmail.com",
    "address": {
        "country": "Bangladesh"
    },
    "bill_to": "self",
    "company": "Rudder",
    "last_name": "Gupta",
    "first_name": "Shivam",
    "preferred_locale": "en-US"
}

Response ::
{
    "id": "onr4t050okt8",
    "object": "account",
    "code": "123456-1251",
    "parent_account_id": null,
    "bill_to": "self",
    "state": "active",
    "username": null,
    "email": "email--927@gmail.com",
    "cc_emails": null,
    "preferred_locale": "en-US",
    "first_name": "Shivam",
    "last_name": "Gupta",
    "company": "Rudder",
    "vat_number": null,
    "tax_exempt": false,
    "exemption_certificate": null,
    "address": {
        "street1": null,
        "street2": null,
        "city": null,
        "region": null,
        "postal_code": null,
        "country": "BD",
        "phone": null
    },
    "billing_info": null,
    "shipping_addresses": [],
    "custom_fields": [],
    "has_live_subscription": false,
    "has_active_subscription": false,
    "has_future_subscription": false,
    "has_canceled_subscription": false,
    "has_paused_subscription": false,
    "has_past_due_invoice": false,
    "created_at": "2021-04-06T07:22:02Z",
    "updated_at": "2021-04-06T07:22:02Z",
    "deleted_at": null
}